File: app/src/main/java/com/felipecsl/elifut/ElifutApplication.java
package com.felipecsl.elifut;

import android.support.multidex.MultiDexApplication;
import com.crashlytics.android.Crashlytics;
import io.fabric.sdk.android.Fabric;

public class ElifutApplication extends MultiDexApplication {
  protected ElifutComponent component;

  @Override public void onCreate() {
    super.onCreate();
    Fabric.with(this, new Crashlytics());
    component = ElifutComponent.Initializer.init(this);
    StethoInitializer.initialize(this);
  }

  public ElifutComponent component() {
    return component;
  }
}

File: app/src/main/kotlin/com/felipecsl/elifut/ElifutApplication.kt
package com.felipecsl.elifut

import android.support.multidex.MultiDexApplication
import com.crashlytics.android.Crashlytics
import io.fabric.sdk.android.Fabric

public open class ElifutApplication : MultiDexApplication() {
  var component: ElifutComponent? = null

  override fun onCreate() {
    super.onCreate()
    Fabric.with(this, Crashlytics())
    component = ElifutComponent.Initializer.init(this)
    StethoInitializer.initialize(this)
  }

  fun component(): ElifutComponent {
    return component!!
  }
}
